ivy_tests/test_ivy/helpers/hypothesis_helpers/dtype_helpers.py

```diff
@@ -31,12 +31,53 @@
 
 
 def _get_fn_dtypes(framework, kind="valid"):
+    """
+    Return a list of supported data types for a specified machine learning framework and kind of data type.
```

Comment:
Let's remove the `machine learning` part, this should also note that it returns the list of the supported dtypes for the current running function specifically.